tests/test_utils_reqser.py

```diff
@@ -2,7 +2,7 @@
 
 from scrapy.http import Request, FormRequest
 from scrapy.spiders import Spider
-from scrapy.utils.reqser import request_to_dict, request_from_dict
+from scrapy.utils.reqser import request_to_dict, request_from_dict, _get_method
```

Comment:
Should we include a "# GROUP 12 ADDED IMPORT" comment here?

Reply:
This is resolved by https://github.com/andnil5/scrapy/pull/32/commits/95dee02acf16b56c662dc104a2a48dec29ad6f27 👍